squid-blockexplorer/src/blocks/utils.ts

```diff
@@ -113,16 +114,25 @@ export function calcSpacePledged(solutionRange: bigint): bigint {
   * @return {bigint} - size of the history in bytes
   */
 export function calcHistorySize(segmentsCount: number): bigint {
-  const KZG_WITNESS_SIZE = 48;
-  const KZG_COMMITMENT_SIZE = 48;
-  const PIECE_SIZE = 32 * 1024;
-  const PIECES_IN_SEGMENT = 256;
-  const RECORD_SIZE = PIECE_SIZE - (KZG_WITNESS_SIZE + KZG_COMMITMENT_SIZE);
-  const RECORDED_HISTORY_SEGMENT_SIZE = (RECORD_SIZE * PIECES_IN_SEGMENT) / 2;
-  const SEGMENT_SIZE =
-    (RECORDED_HISTORY_SEGMENT_SIZE / RECORD_SIZE) * PIECE_SIZE * 2;
-
-  return BigInt(segmentsCount * SEGMENT_SIZE);
+
+  const SLOT_PROBABILITY = BigInt(1) / BigInt(6);
```

Comment:
When working with integers (like here) `1/6` is always `0`, dividing by `0` in the formula below will always result in either NaN or Infinity or exception/error. See how division is done in monorepo codebase where we treat numerator and denominator as separate numbers for that exact reason.